frontend/app/controllers/skill-search.js

```diff
@@ -3,7 +3,6 @@ import { action, computed } from "@ember/object";
 import { observes } from "@ember-decorators/object";
 import { inject as service } from "@ember/service";
 import Controller from "@ember/controller";
-import PeopleSkill from "../models/people-skill";
 
 @classic
 export default class SkillSearchController extends Controller {
```

Comment:
Okay so about this. You've probably noticed there is a looooot of repeated code here, that's not the idea.

My idea would be:
- The SkillSearchController has an array. There is one Object in the array by default. 
- The object has two attributes: level, skill
- Every time you press the "Skill hinzufügen" Button, an object gets added to the array
- In the skill-search.hbs: loop over the array and for every object, show a component.
- That component has the slider and powerselect for one level and one skill. These are the ones you pass when iterating over the array
- If a value gets changed, bubble an action to the SkillSearchController.
- That action gets the values of all the filter components, these should be in the array. And then create the backend call with all the data inside the Controller